src/components/searchbox.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import elasticsearch from 'elasticsearch'
+
+let client = new elasticsearch.Client({
+	host: 'localhost:9200',
```

Comment:
In real scenario, client app would look at some host instead of localhost (which is fine for development).